src/style.css

```diff
@@ -1,3 +1,89 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  list-style: none;
+}
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Style all elements using '.class'. For example, instead of styling 'ul', give it a class and style that class.